python/GafferUI/SpreadsheetUI.py

```diff
@@ -1833,7 +1833,7 @@ def __addToSpreadsheet( plug, spreadsheet, sectionName = None ) :
 		columnIndex = __addColumn( spreadsheet, plug )
 		if sectionName is not None :
 			_SectionChooser.setSection(
-				spreadsheet["rows"]["default"]["cells"][columnIndex].source(),
+				spreadsheet["rows"].defaultRow()["cells"][columnIndex].source(),
```

Comment:
Should we update the other calls to `rowsPlug["default"]` in here? As well as being consistent, it removes the name lookup overhead right?

Reply:
Indeed - thanks for catching that. 4ee88a2.